test/sql/attach/attach_export_import.test

```diff
@@ -7,16 +7,39 @@ require skip_reload
 statement ok
 ATTACH ':memory:' AS db1
 
+statement ok
+USE db1;
+
 statement ok
 CREATE TABLE db1.integers(i INTEGER);
 
 statement ok
 INSERT INTO db1.integers VALUES (1), (2), (3), (NULL);
 
+# FIXME: when we don't use 'USE' then we have to refer to 'integers' as 'db1.integers'
+# this breaks when re-imported, because the table will be created as just 'integers', not 'db1.integers'
```

Comment:
This can likely be fixed in a future PR.
Since we serialize the dependencies of the view, we can traverse the QueryNode and replace the references made to tables/macros/etc within the same catalog.

i.e this would replace `db1.integers` to `memory.integers` (the catalog that the view gets created into it)